### ISSUE TITLE
ui: table view sorts lexicographically when it should sort numerically

### ISSUE THREAD

**Issue by mark-rushakoff** [influxdata/influxdb#13526]
Here is a screenshot of my raw data. Note that the rightmost numeric column is of type `long`.

![Screen Shot 2019-04-19 at 3 25 54 PM](https://user-images.githubusercontent.com/290647/56446439-874cb800-62b7-11e9-9b24-2601efee0a4d.png)

When rendered as a table, the column appears to be sorted by the string representation of the numbers rather than by the numeric values:

![Screen Shot 2019-04-19 at 3 24 42 PM](https://user-images.githubusercontent.com/290647/56446459-aa776780-62b7-11e9-90fe-1c24d466c54d.png)

Running with build 11bfc0b.

**Comment by mark-rushakoff** [comment]
I assume based on #14217 that this issue is still outstanding.

**Comment by lukevmorris** [comment]
Reopening this

Flux table result:

```csv
#group,false,false,false
#datatype,string,long,long
#default,_result,,
,result,table,bytes
,,0,132800339
,,0,39665590
,,0,8505848
,,0,6301539
,,0,3709228
,,0,1491019
,,0,1301411
,,0,1279743
,,0,862723
,,0,757414
,,0,328315
,,0,25423
,,0,22234
,,0,22030
```

When I view the `Raw Data`, I see the flux table as it appears above.

<img width="446" alt="Screen Shot 2019-11-19 at 12 11 29 PM" src="https://user-images.githubusercontent.com/1438089/69182385-c0efd680-0ac5-11ea-9d72-cdf5c8c3ee50.png">

When I switch to a `Table` graph type, I see the following:

<img width="1109" alt="Screen Shot 2019-11-19 at 12 08 04 PM" src="https://user-images.githubusercontent.com/1438089/69182159-4f178d00-0ac5-11ea-8b0a-1ffebc2b16bd.png">
<img width="1111" alt="Screen Shot 2019-11-19 at 12 08 11 PM" src="https://user-images.githubusercontent.com/1438089/69182160-4f178d00-0ac5-11ea-99d5-5d6ff0552d97.png">

These appear to be sorted alphabetically instead of numerically.

**Comment by lukevmorris** [comment]
Related: https://github.com/influxdata/influxdb/issues/15980

**Comment by rawkode** [comment]
Just bumped into this problem too. Even though my Flux has `sort()`, the table view does it's own sorting with string comparison rather than numeric.

**Comment by Asalem1** [comment]
This should be resolved in #16235
